recipes/fpdf/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "fpdf" %}
+{% set version = "1.7.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fpdf-{{ version }}.tar.gz
+  sha256: 125840783289e7d12552b1e86ab692c37322e7a65b96a99e0ea86cca041b6779
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - python >=3
+
+test:
+  imports:
+    - fpdf
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://code.google.com/p/pyfpdf
+  summary: Simple PDF generation for Python
+  license: LGPLLR
```

Comment:
This file tells me it's regular LGPL v3? https://github.com/reingart/pyfpdf/blob/master/LICENSE

Reply:
Hm apparently. Fixing.